COPYING

```diff
@@ -1,6 +1,6 @@
 # License 1
 
-Copyright (c) 2024-present, Valkey contributors
+Copyright (c) 2024-present, Redis contributors
```

Comment:
@madolson shouldn't this be Valkey here? This is the license with "the copyright holders" in the 3rd clause.

License 2 below is with "Redis" in the 3rd clause.

Reply:
Correct

Reply:
```suggestion
Copyright (c) 2024-present, Valkey contributors
```